electron/src/main/index.js

```diff
@@ -8,113 +8,104 @@
 
 'use strict'
 
-import {app, BrowserWindow, ipcMain, dialog, shell} from 'electron'
-var fs = require('fs');
-var path = require('path');
-var tmp = require('tmp');
-
+const path = require('path')
+const fs = require('fs')
+const tmp = require('tmp')
 const url = require('url')
+const { app, BrowserWindow, ipcMain, dialog, shell, Menu} = require('electron')
+// url for printPDF flask server which is used in development and production mode
 
-/**
- * Set `__static` path to static files in production
- * https://simulatedgreg.gitbooks.io/electron-vue/content/en/using-static-assets.html
- */
-if (process.env.NODE_ENV === 'development') {
-  // we were run as electron --inspect=5858 path/to/main.js <args>
-  // so get rid of the first two args
-  console.log("args " + process.argv)
-  process.argv.shift()
-  process.argv.shift()
-} else {
-  global.__static = path.join(__dirname, '/static').replace(/\\/g, '\\\\')
-}
+var port = process.env.FLASKPORT
+const printPdfUrl = `http://127.0.0.1:${port}/`
 
-let mainWindow
+const isDev = process.env.BABEL_ENV === 'development'
 
-var target = process.argv[1] || "";
+var target = null
+// Finds this url in the argv array and sets to target value
+if (process.argv.includes(printPdfUrl)) {
+    target = process.argv.find(element => element === printPdfUrl);
```

Comment:
Why do we need to find the value here?  Why not `target = printPdfUrl`?

Reply:
Right, I did overthinking this and target = printPdfUrl works.